pkg/aws/apis/validation/validation.go

```diff
@@ -219,3 +242,11 @@ func contains(arr []string, checkValue string) bool {
 	}
 	return false
 }
+
+func validateStringValues(fld *field.Path, s string, accepted []string) field.ErrorList {
```

Comment:
I have seen this kind of function too many times. One could just use https://pkg.go.dev/golang.org/x/exp/slices which essentially contains a lot of such utilities and prevents duplication of code. For now you can continue using this.

Reply:
I changed to use the `slices` package. I wasn't aware of it but we can use that instead.